box-content-sdk/src/main/java/com/box/androidsdk/content/models/BoxJsonObject.java

```diff
@@ -24,8 +22,7 @@
 
     private static final long serialVersionUID = 7174936367401884790L;
     // Map that holds all the properties of the entity. LinkedHashMap was chosen to preserve ordering when outputting json
-    protected JsonObject mJsonObject;
-    transient CacheMap mCacheMap;
+    CacheMap mCacheMap;
```

Comment:
We need to move the same read and write logic to cache. There is no need to restore the map but we need to initialize the same in read.